### ISSUE TITLE
Useless dependency on future

### ISSUE THREAD

**Issue by frenzymadness** [FirebirdSQL/fdb#115]
Hi. I see future in dependencies:
https://github.com/FirebirdSQL/fdb/blob/27e0f25c45a8310f8b657c5e08cdd6e656f92970/setup.py#L55
but I cannot find a single usage of it in the codebase so I think it can be removed.

**Comment by mberdyshev** [comment]
The issue is outdated now. Nevertheless, I find the latest dependencies useless as well, cause I was not able to find any usage.

**Comment by frenzymadness** [comment]
You are right that the `future` is no longer there. Thank you. I see you've created #124, so I'm gonna close this one.
